.github/workflows/github-ci.yml

```
@@ -21,14 +21,17 @@ jobs:
         run: npm install
 
       - name: ESLint
-        run: npm run eslint
+        run: npm run build:lint
+
+      - name: Prettier
+        run: npm run build:prettier:check
 
       - name: Upload Artifacts
         uses: actions/upload-artifact@v3
```

actionlint (1.7.12)

[error] the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)